databases/week2/Week2HWTASK.sql

```diff
@@ -0,0 +1,56 @@
+-- Part 1: Working with tasks
+-- Add a task
+select * from task;
+INSERT INTO task 
+VALUES 
+(36,"Do MYSQL HYF exercises","Read the exercises and resolve it","2018-10-10 10:30:05","2018-10-09 10:30:05","2018-10-20 11:30:05",1);
+
+ -- Change the title of a task
+ALTER TABLE task RENAME COLUMN title TO task_title;
+UPDATE task set task_title = "Play music"  WHERE id=11;
+-- Change a task due date
+UPDATE task set due_date = "2017-11-20 23:58:03" WHERE id=2;
+-- Change a task status
+UPDATE task set status_id = 3 WHERE id=1;
+-- Mark a task as complete
+UPDATE task set status_id = 3 WHERE task_title="Charge electric bicycle" AND id=22;
```

Comment:
If you are using the `id` here to select a task there is no need to use the task title as well, the `id` is unique ;)

Reply:
Valid point !!